### ISSUE TITLE
Sublime instructions

### ISSUE THREAD

**Issue by whitequark** [AltGr/opam-user-setup#4]
@gasche asked me to describe what setup Sublime Text requires, so here it goes.

  * Sublime has its own package manager, [Package Control](http://sublime.wbond.net)
  * Using Package Control, the user should install [updated OCaml highlighting](https://packagecontrol.io/packages/OCaml), [ocp-index integration](https://packagecontrol.io/packages/OCaml%20Autocompletion) or [merlin integration](https://packagecontrol.io/packages/Merlin).

The packages are installed by placing them into `~/.config/sublime-text-X/Installed Packages`, but I don't know how to download them from http://sublime.wbond.net.

**Comment by AltGr** [comment]
Thanks, I'll try to get to Sublime next.

**Comment by AltGr** [comment]
Ok, I've experimented with this, and have a very preliminary version that installs ST files, but it seems to me that the current bindings for ST are not mature enough to be worth adding at the moment:
* there is no ocp-indent binding
* the merlin binding is still at an alpha stage, and it's page mentions

    > This is an initial port of the vim plugin of merlin. Buffers can be synchronised with merlin, but any edit needs a full refresh between ST and merlin. Projects are not currently supported.
* I tried the ocp-indent plugin with sublime-text 2, but it didn't load out-of-the-box using my version of Python (no `shutil.which`) ; qualified completions where shown last (`Sys.f` proposes `for` and many others before `Sys.file_exists`), and worse, the module name is removed upon completing, which makes it unusable at the moment.

For these reasons, it seems reasonable to let ST users install from their package manager if they want to experiment, rather than integrate an half-baked solution.

**Comment by whitequark** [comment]
The first two are correct. The third one obviously is not, I was using sublime-ocp-index for several months. It's probably not compatible with ST2; try ST3. (http://sublimetext.com/3)

**Comment by AltGr** [comment]
Right, that was on ST2 ; if it works well on ST3 that's good news ! I'll try it, thanks.

**Comment by AltGr** [comment]
Is there any chance you might get the motivation to write an ocp-indent plugin ? :D

**Comment by whitequark** [comment]
Probably not, I've never used ocp-indent and I always indent my files by hand. Even if I write it, not using the code you write is a recipe for disaster.

**Comment by whitequark** [comment]
Merlin on ST3 is now quite stable and useful so this should be revisited, I think.
